src/storage/src/region/tests.rs

```diff
@@ -314,8 +315,13 @@ async fn test_recover_region_manifets() {
     builder.root(&tmp_dir.path().to_string_lossy());
     let object_store = ObjectStore::new(builder).unwrap().finish();
 
-    let manifest =
```

Comment:
Can we add a test for `compress=true` for region recovery? I think we can change this function into a normal function that accepts the boolean argument `compress`:

 ```rust
async fn test_recover_region_manifets(compress: bool) {
     ...
    let manifest = if compress {
           ...
    } else {
         ...
    };
     ....
}
```

And adds two tests:


```rust
#[tokio:test]
async fn test_recover_region_manifets_compress() {
    test_recover_region_manifets(true);
}

#[tokio:test]
async fn test_recover_region_manifets_uncompress() {
    test_recover_region_manifets(false);
}
```